test-apps/imodel-transformer/src/IModelHubUtils.ts

```diff
@@ -48,7 +48,7 @@ export class IModelTransformerTestAppHost {
         + "Please see the .env.template file on how to set up environment variables."
       );
       return new Promise<AccessToken>(async (resolve, reject) => {
-        const client = await ElectronMainAuthorization.create({
+        const client = new ElectronMainAuthorization({
```

Comment:
why is this change to ElectronMainAuthorization not followed up by `signInSilent` like the rest?
Is it because of the lazy sign in?

Reply:
pulled and tested and access token retrieval still works fine here. I do locally get 
`Unauthorized: The user is unauthorized. Please provide valid authentication credentials.`
but the access token is valid so it should be unrelated afaict. Going to resolve this and look into why my access token is not authorizing me for the itwin I tested on separately.